src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,161 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node head;
+    private Node tail;
+    private int size;
+
+    private class Node {
```

Comment:
[Inner class should be at the end of the class.](https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md#make-nodet-an-inner-class)
```suggestion
    private static class Node {
```

create constructor in class Node